packages/common/src/api/utils.ts

```diff
@@ -2,6 +2,9 @@ import { z } from 'zod'
 
 import { decodeHashId, encodeHashId } from '~/utils/hashIds'
 
+// TODO: Move usage of these utils
+// to the version in packages/common/src/models/Identifiers.ts
+// https://linear.app/audius/issue/PAY-2997/migrate-idhashid-usage-to-models-version
```

Comment:
Makes sense - should we add this to the migrations list somewhere? Just afraid we'll forget about this